packages/deno/src/executors/esbuild/schema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "version": 2,
+  "title": "Deno Emit Executor",
+  "description": "Output a single JavaScript file with all dependencies",
+  "type": "object",
+  "properties": {
+    "denoConfig": {
+      "type": "string",
+      "description": "Path to the Deno configuration file."
+    },
+    "main": {
+      "type": "string",
+      "description": "The entry point for your Deno application."
+    },
+    "outputFile": {
+      "type": "string",
+      "description": "The destination of the emitted JavaScript file."
+    },
+    "bundle": {
+      "type": "boolean",
+      "description": "Bundle all imports into a single JavaScript file.",
+      "default": true
+    },
+    "assets": {
+      "type": "array",
+      "description": "List of static assets.",
+      "default": [],
+      "items": {
+        "$ref": "#/definitions/assetPattern"
+      }
+    },
+    "sourceMap": {
+      "oneOf": [
+        { "type": "string", "enum": ["linked", "inline"] },
+        { "type": "boolean", "enum": [false] }
```

Comment:
probably just change this to 
`{ "type": "boolean" }`
and if it's true set it sourceMap type to "inline". just a bit nicer DX.